programs/protocol-contracts-solana/src/lib.rs

```diff
@@ -350,8 +350,9 @@ pub struct Withdraw<'info> {
 
     #[account(mut)]
     pub pda: Account<'info, Pda>,
+    /// CHECK: to account is not read so it's not dangerous
```

Comment:
i'll keep it simple in the comment, but expand here: 

`Account<'info, T>` is an Anchor convenient struct that automatically does "ownership" check, which `UncheckedAccount` explicitly does not. 

Ownership is usually critical because these accounts are completely supplied by user (tx builder) and
could be anything. The onwership validation makes sure the user supplied the correct account. 
For example, in the above `pub pda: Account<'info, Pda>`, Anchor does ownership check 
`Account. info. owner == T::owner()`, which checks whether the supplied PDA account is in fact
owned by the current program (i.e. created by the current program). This is crucial otherwise
the program would be directed to read any user supplied potential malicious PDA account. 

For the `to` field, obviously we don't care who its owner is because we don't read it. Whatever
tx builder tells us to send to, we happily send to. In this case Anchor requires program to explicit
exempt ownership checking by the `UncheckedAccount` struct and also the comment` ///CHECK`.